src/View/Item/showItem.html.twig

```diff
@@ -55,6 +55,37 @@
                 <h2>D'autres produits de la catégorie qui pourraient vous intéresser</h2>
                 <h3>{{ items[0].category }}</h3>
             </div>
+
+            <div class="card col-lg-4 col-sm-8 col-md-4 mx-auto">
```

Comment:
non, il faut faire un boucle 
{%for itemRandom in itemsRandom %}
...
et comme ça tu n'as pas à répéter 3 fois le code HTML ;-)